src/index.ts

```diff
@@ -37,14 +38,20 @@ async function main() {
 		headless: Config.browser.isHeadless
 	});
 
+	/* eslint-disable no-await-in-loop */
 	for (const store of Stores) {
 		Logger.debug(store.links);
 		if (store.setupAction !== undefined) {
 			store.setupAction(browser);
 		}
 
+		if (store.linksBuilder) {
+			await fetchLinks(store, browser);
+		}
+
 		setTimeout(tryLookupAndLoop, getSleepTime(), browser, store);
 	}
+	/* eslint-enable no-await-in-loop */
```

Comment:
I think you're fine using [this solution](https://eslint.org/docs/rules/no-await-in-loop)

```suggestion
	const promises = [];
	for (const store of Stores) {
		Logger.debug(store.links);
		if (store.setupAction !== undefined) {
			store.setupAction(browser);
		}
		
		if (store.linksBuilder) {
			promises.push(fetchLinks(store, browser));
		}

		setTimeout(tryLookupAndLoop, getSleepTime(), browser, store);
	}
	await Promise.all(promises)
```

Reply:
Have pushed a change using `Promise.all(...)`.